src/content/docs/en/pages/store-journey/sql/edgesql-shell-commands.mdx

```diff
@@ -0,0 +1,277 @@
+
+```sh
+EdgeSQL> help .import
+```
+
+You will receive a response similar to this:
+
+```sh
+.import: Import data from file files, Kaggle datasets, or databases into a database table.
+
+    Args:
+        arg (str): The import command and its arguments.
+
+    Command Formats:
+        .import file <csv|xlsx> <file_path> <table_name>: Import data from a file CSV or Excel file.
+        .import kaggle <dataset> <data_name> <table_name>: Import data from a Kaggle dataset.
+        .import mysql <database> <source_table> <table_name>: Import data from a MySQL database table.
+        .import postgres <database> <source_table> <table_name>: Import data from PostgreSQL database table.
+        .import turso <database> <source_table> <table_name>: Import data from Turso database.
+
+    Examples:
+        .import file csv /path/to/file.csv my_table
+        .import kaggle joonasyoon/google-doodles list.csv list
+        .import mysql mydb_name source_table_name my_table
+        .import turso <database> <source_table> <table_name>
+```
+
+### .dbsize
+
+Gets the size of the current database in MB.
+
+```sh
+EdgeSQL> .dbsize
+```
+
+### .exit
+
+Exits the EdgeSQL Shell.
+
+```sh
+EdgeSQL> .exit
+```
+
+## Environment variables
+
+When using the EdgeSQL Shell to import data from external sources, you can send the credentials and tokens as environment variables byt using the `export` command. This command is also used for setting your Azion **personal token**.
+
+### Personal token
+
+You can send your **personal token** as an environment variable in EdgeSQL Shell.
+
+```sh
+ export AZION_TOKEN="<your_auth_token_here>"
+```
+
+### Kaggle credentials
+
+Use the following commands to set your Kaggle credentials:
+
+```sh
+  export KAGGLE_USERNAME="<username>"
+  export KAGGLE_KEY="<kaggle_api_key>"
+```
+
+### MySQL credentials
+
+Use the following commands to set your MySQL credentials:
+
+```sh
+  export MYSQL_USERNAME="<username>"
+  export MYSQL_PASSWORD="<password>"
+  export MYSQL_HOST="<host_address>"
+```
+
+Optional settings for MySQL:
+
+```sh
+  export MYSQL_PORT=<port>
+  
+  # For TLS connection
+  export MYSQL_SSL_CA="<ssl_ca>"
+  export MYSQL_SSL_CERT="<ssl_cert>"
+  export MYSQL_SSL_KEY="<ssl_key>"
+  export MYSQL_SSL_VERIFY_CERT=<True|False>
+```
+
+### PostgreSQL credentials
+
+Use the following commands to set your PostgreSQL credentials:
+
+```sh
+  export POSTGRES_USERNAME="<username>"
+  export POSTGRES_PASSWORD="<password>"
+  export POSTGRES_HOST="<host_address>"
+```
+
+Optional settings for PostgreSQL:
+
+```sh
+  export POSTGRES_PORT=<port>
+  
+  # For TLS connection
+  export POSTGRES_SSL_CA="<ssl_ca>"
+  export POSTGRES_SSL_CERT="<ssl_cert>"
+  export POSTGRES_SSL_KEY="<ssl_key>"
+  export POSTGRES_SSL_VERIFY_CERT=<True|False>
+```
+
+### Turso credentials
+
+Use the following commands to set your Turso credentials:
+
+```sh
+ export TURSO_DATABASE_URL=<https://<db_name>-<organization>.turso.io
```

Comment:
Please, see if this alternative is correct: export TURSO_DATABASE_URL=https://<db_name>-<organization>.turso.io>